packages/server/src/threads/automation.ts

```diff
@@ -625,6 +625,7 @@ export async function executeInThread(job: Job<AutomationData>) {
   })
 
   return await context.doInAppContext(appId, async () => {
+    await context.ensureSnippetContext()
```

Comment:
External triggers are the only automation type that I can see where we don't wrap them in automation context, so I'm manually adding them here (just like the other contexts are manually added here).